code/modules/multiz/core.dm

```diff
@@ -1,47 +1,47 @@
-/proc/get_step_multiz(ref, dir)
-	if(dir & UP)

```

Comment:
Line ending fuckup?

Reply:
moved the stuff to modules/multiz/_core.dm

Reply:
The line endings or something is still changed though, if you move a file with no changes it wouldn't display this.